customersupport/controllers/ordersearch.py

```diff
@@ -42,7 +40,7 @@ def select_search():
             mock=False
         )
 
-    # Call the Sales API to get matching . Used by the search-order
+    """Call the Sales API to get matching . Used by the search-order"""
```

Comment:
@eyoon13 I think I may have confused you with my note about comments.
Normal in-line comments should be:
```python
# This is a comment.
```

The type of comments that use the `"""` version is when making function doc. strings. I recommend reading [this PEP guide](https://www.python.org/dev/peps/pep-0257/) to understand how doc. strings work.
For a function, it would look like this:
```python
def kos_root():
    """Return the pathname of the KOS root directory."""
    global _kos_root
    if _kos_root: return _kos_root
    ...
```